rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/model/EjbSessionBeanModel.java

```diff
@@ -80,11 +80,11 @@
      * Contains the jndi location for this resource.
      */
     @Adjacency(label = JNDIResourceModel.TYPE, direction = Direction.OUT)
-    public JNDIResourceModel getJndiReference();
+    public Iterable<JNDIResourceModel> getJndiReferences();
 
     /**
      * Contains the jndi location for this resource.
      */
     @Adjacency(label = JNDIResourceModel.TYPE, direction = Direction.OUT)
-    public void setJndiReference(JNDIResourceModel jndiReference);
+    public void addJndiReference(JNDIResourceModel jndiReference);
```

Comment:
Or, we can have a collection of EJBJNDIReference and this will contain such information in enum. It will be more scalable (in case there will be some other reference introduced by other servers or I don't know), WDYT?